my_list.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*numbers)
+    @list = numbers
+  end
+
+  def to_s
+    @list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- [ ] I'll suggest you to implement your `each` method as below. In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.
the code should resemble this:
```
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```